Improve missing certs/pki error

Right now when somebody attempts to get started with Aurae they are often missing the TLS cert material. The error does not give the user a path forward or offer a resolution. Can we please improve the error message? Maybe point the user to a low-risk URL that will always have an answer?

```
2023-01-15T12:13:22.256096Z  INFO auraed::init::system_runtimes::daemon_system_runtime: Running as a daemon.
2023-01-15T12:13:22.303720Z  INFO auraed::init::system_runtimes: User Access Socket Created: /var/run/aurae/aurae.sock
2023-01-15T12:13:22.342598Z ERROR auraed: Err(Failed to read server certificate: /etc/aurae/pki/_signed.server.crt

Caused by:
    No such file or directory (os error 2))
make: *** [Makefile:150: start] Error 1
```

Maybe we can improve it by adding something such as:

```
Aurae requires a signed TLS certificate to run as a server. Please see <URL> for information on best practices to quickly generate one.
```

--- a/auraed/src/lib.rs
+++ b/auraed/src/lib.rs
@@ -230,7 +230,9 @@ impl AuraedRuntime {
         let server_crt =
             tokio::fs::read(&self.server_crt).await.with_context(|| {
                 format!(
-                    "Failed to read server certificate: {}",
+                    "Aurae requires a signed TLS certificate to run as a server, but failed to 
+                    load: '{}'. Please see https://aurae.io/certs/ for information on best 
+                    practices to quickly generate one.",
                     self.server_crt.display()
                 )
             })?;

